File: calc.c
/* Name: Mariyam Zagral
     UIN:251P084
     Class: Comp / D */

#include<stdio.h>

int main() {

int choice;
 float num1, num2, result;

start:

printf("\n-- Simple Calculator ---\n");
printf("1. Addition\n");
printf("2. Subtraction\n");
 printf("3. Multiplication\n");
printf("4. Division\n");
printf("5. Exit\n");

printf("Enter your choice: ");
scanf("%d", &choice);


if (choice == 5) {
printf("Exiting...\n");
return 0;

}
printf("Enter two numbers: ");
scanf("%f %f", &num1, &num2);

switch (choice) {

case 1:
result =num1 + num2;
printf("Result = %.2f\n", result);
break;

case 2:
result = num1- num2;
printf("Result = %.2f\n", result);
break;

case 3:
result=num1*num2;
printf("Result = %.2f\n", result);
break;

case 4:
if (num2 == 0)
printf("Cannot divide by zerol\n");
else {
result = num1 / num2;
printf("Result = %.2f\n", result);
}
break;

default:
printf("Invalid choice!\n");

 }
goto start;

 }

File: calc.py
# Name:Mariyam Zagral
# UIN: 251P084
print("Mariyam Zagral")
print("UIN: 251P084")
a = float(input("Enter first number: "))
b = float(input("Enter second number: "))

def add():
    print("Result:", a + b)

def subtract():
    print("Result:", a - b)

def multiply():
    print("Result:", a * b)

def divide():
    if b == 0:
        print("Error! Division by zero is not allowed.")
    else:
        print("Result:", a / b)



print("Choose operation:")
print("1. Addition (+)")
print("2. Subtraction (-)")
print("3. Multiplication (*)")
print("4. Division (/)")

choice = input("Enter choice (1/2/3/4): ")

if choice == '1':
    add()

elif choice == '2':
    subtract()

elif choice == '3':
    multiply()

elif choice == '4':
    divide()

else:
    print("Invalid choice!")
